celadon_ivi/mixins.spec

```diff
@@ -23,15 +23,15 @@ allow-missing-dependencies: true
 dexpreopt: true
 pstore: ram_dummy(address=0x50000000,size=0x400000,record_size=0x4000,console_size=0x200000,ftrace_size=0x2000,dump_oops=1)
 bugreport: true
-media: mesa(add_sw_msdk=false, opensource_msdk=true, opensource_msdk_omx_il=true)
+media: mesa(enable_msdk_omx=false, add_sw_msdk=false, opensource_msdk=true, opensource_msdk_omx_il=false)
```

Comment:
shall we change the ivi config? they will stay at broxton (GP) and using omx and msdk?

Reply:
sounds good to me. thanks.

Reply:
Sure, it's good.